clientlibs/js/src/ApiService/ApiService.types.ts

```diff
@@ -0,0 +1,22 @@
+import { UpGradeClientEnums } from 'types';
+
+export interface IApiServiceRequestParams {
+  path: string;
+  method: UpGradeClientEnums.REQUEST_METHOD;
+  options?: any;
```

Comment:
I don't see the param 'options' used anywhere; `sendRequest()`  takes the other three params and creates 'options' from `path`. Is it needed for some other reason?

Reply:
👀 maybe not, I'll look

Reply:
was leftover from a refactor, removed